scripts/upgrade-pwa.js

```diff
@@ -6,8 +6,8 @@ const pinned = {
   '@ng-bootstrap/ng-bootstrap': '11', // 12 requires Bootstrap 5
   '@types/node': '16', // LTS
   '@cspell/dict-de-de': '1.1.32', // later versions use the GPL license
-  jest: '27',
-  'jest-preset-angular': '11',
+  jest: '28',
+  'jest-preset-angular': '12',
```

Comment:
Is it still necessary or advisable to pin these dependencies? Aren't these the latest versions? Do we expect problems with newer ones?